travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls Click oca-pylint-plugin > /dev/null 2>&1
```

Comment:
I'm agree with you.
You can validate now.

Similar case as you validate other component of this project.

By example how you validate `pylint` or `flake8`?
You don't avoid the use because don't is in OCA project.

Then you can validate with result of unitest 
[![Build Status](https://travis-ci.org/moylop260/pylint_oca.svg?branch=master)](https://travis-ci.org/moylop260/pylint_oca)
 [![Coverage Status](https://coveralls.io/repos/moylop260/pylint_oca/badge.svg?branch=master&service=github)](https://coveralls.io/github/moylop260/pylint_oca?branch=master)

Custom local test.

No is the first time that we use a external library that don't is in OCA.